src/components/input.js

```diff
@@ -366,6 +366,7 @@ const withAutocomplete = (WrappedComponent) =>
             clearSelectionRef.current = clearSelection;
             return div(
               {
+                'aria-controls': 'autocomplete-suggestions', // Required for screen readers to announce the suggestions container
```

Comment:
this does not seem right. Is this absolutely necessary? By inspecting the DOM, I can see:
* the downshift library already sets `aria-controls="downshift-0-menu"` on the input element. You're adding an additional `aria-controls` on the div containing the input element.
* the element containing the autosuggestions has an id of `downshift-0-menu`, not `autocomplete-suggestions`

_aria-controls attributes:_
![Screenshot 19-09-2024 at 10 22 (2)](https://github.com/user-attachments/assets/493e1857-4747-493d-b416-54c31fabce51)

_autosuggest options_:
![Screenshot 19-09-2024 at 10 25](https://github.com/user-attachments/assets/91ebb5c0-db97-48b8-8711-0da5b32ba1c6)

Reply:
As explained, the Downshift component has WrapperComponent that has an attribute `role='listbox'` which requires then `aria-controls=<DownshiftMenuId>`. I have [updated](https://github.com/DataBiosphere/terra-ui/pull/5092/commits/54ca08fc0142289452eece583b3e50757b4a0a42) the **DownshiftMenuId** to reference the dynamic MenuId when opened or the WrapperId when closed.